src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAResolvePackageAssetsTask.cs

```diff
@@ -117,6 +117,80 @@ public void It_does_not_error_on_duplicate_package_names()
             new CacheWriter(task); // Should not error
         }
 
+        [Fact]
+        public void It_warns_on_incorrectly_cased_culture_codes_of_resources()
+        {
+            string projectAssetsJsonPath = Path.GetTempFileName();
+            var assetsContent = @"
+{
+  `version`: 3,
+  `targets`: {
+    `net7.0`: {
+      `JavaScriptEngineSwitcher.Core/3.3.0`: {
+        `type`: `package`,
+        `compile`: {
+          `lib/netstandard2.0/JavaScriptEngineSwitcher.Core.dll`: {}
+        },
+        `runtime`: {
+          `lib/netstandard2.0/JavaScriptEngineSwitcher.Core.dll`: {}
+        },
+        `resource`: {
+          `lib/netstandard2.0/ru-ru/JavaScriptEngineSwitcher.Core.resources.dll`: {
+            `locale`: `what is this even`
```

Comment:
Looks like the test names and data are backwards, the test name here seems to indicate that the locale for this test should be `ru-ru`, and the current value swapped to the subsequent test.